.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Node CI
+
+on: 
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
Why do you need a matrix for one version?

Reply:
Easier to change/add versions this way